src/popup/qrcode.css

```diff
@@ -117,3 +117,17 @@ html, body {
   /* override inerhited from browser-style */
   margin: 0px;
 }
+
+/* shift popup colors if user has specified dark mode (reads preference from OS, not browser) */
+@media (prefers-color-scheme: dark) {
+  .qrcode {
+    background-color: var(--grey-30);
+  }
+  #qrcode-container {
+    background-color: var(--grey-60);
```

Comment:
Oh another big problem. This change overrides the user's choice of the background color in the settings:
 
![image](https://user-images.githubusercontent.com/11966684/58823732-b1c1bd00-863a-11e9-9b37-b50efc6240b0.png)

leads to:

![image](https://user-images.githubusercontent.com/11966684/58823752-bab28e80-863a-11e9-8bd7-b5646df42c4a.png)

---

So I am sorry, but it seems you need to adjust some JS-things, too…
(best solution would IMHO be that it only colors the background of the QR code dark, if you want it.)

So, hmm, I guess we need another change in one module even, where we get the default values from: https://github.com/TinyWebEx/AddonSettings/issues/6

(see also the Readme, it provides an extensive doc on how it's used.)



That color is set in JS.